src/index.ts

```diff
@@ -36,6 +36,20 @@ async function main() {
 
   console.log(`This script will payout from block ${minimumHeight} to maximum height ${maximumHeight}`);
 
+  const missingHeights = await getHeightMissing(minimumHeight, maximumHeight);
```

Comment:
Shall we at least move this into queries.ts and hide it there, so alternate implementations can either deal with it, or ignore it, but we don't leak this up to index. (Or, let it be up at index, but put the "business logic" down in queries wrapped up in something like validateConsistency()?)

Reply:
I've started work on #13 so things will be moved around quite a bit. We won't be able to have this for other data sources, but I still like the sanity check for the archivedb. Because of the reorg, this should go in after 13 and it will be more obvious where it fits.